### PR TITLE
Add the ability to Suspend Users

### PR DESCRIPTION
**Pull request by LogischJo**
This allows Admins to suspend User Accounts. An suspended account gets logged out in the next request and an information message gets displayed (on the login page). 
Refers to #204

### CODE REVIEW

**Comment by LogischJo** [comment]
It is inspired by the suspension system of servers because you don't have to reinvent the system every time 😛

**Comment by LogischJo** [comment]
Here some screenshots:

**Comment by anondev-sudo** [comment]
Good

**Comment by anondev-sudo** [comment]
Can you contribute also on referral system or egg/node linking to products if you have times?

**Comment by LogischJo** [comment]
> Can you contribute also on referral system or egg/node linking to products if you have times?

This PR is dedicated to User Suspension, but am I working with AVMG and others on the big new linking feature for Products with nodes and eggs.. Some news about this soon hopefully. The referral feature is an cool idea too but as of now it is not planned by me, but who knows if I have time for it soon

**Comment by LogischJo** [comment]
AVMG and the other devs just reported me back that it would make more sense to suspend all the current running servers from an user.  I really forgotten that, I will look into it tomorrow and then this should be ready to get merged

**Comment by 1day2die** [comment]
working fine

File: app/Models/User.php [hunk 1]
```diff
@@ -189,6 +193,12 @@ public function suspend()
      */
     public function unSuspend()
     {
+        foreach ($this->servers as $server){
+          if ($this->credits >= $server->product->getHourlyPrice()) {
+            $server->unSuspend();
+        }
```
Comment (BitSec01) [inline app/Models/User.php:199]: Make sure the formatting is correct when pushing a PR to keep things clean and have the brackets aligned.
Seems like you need to select line 198 and 199 and press tab and it should be fixed.

Comment (LogischJo) [reply app/Models/User.php]: Got fixed in the new commit 👍

**Review by AVMG20 — APPROVED** [review]

**Comment by BolverBlitz** [comment]
If I see that correctly, it also includes an API call?

**Pull request merged**